src/app/(nousConnaitre)/dateCles/page.js

```
@@ -2,200 +2,202 @@
+          {/* Première division 2014 */}
+          <div className="flex flex-col items-center">
+            <div className="relative ml-[7.5rem] md:ml-[7.5rem] sm:ml-[5rem] xs:ml-[3.7rem] md:mb-8 sm:mb-0 xs:mb-2">
+              <img
+                src="/dateCles/site bab rayan dates clés-02.png"
+                alt="element"
+                className="w-[30rem] h-[15rem] md:w-[30rem] md:h-[15rem] sm:w-[20rem] sm:h-[10rem] xs:w-[15rem] xs:h-[7.5rem]"
+              />
+            </div>
+            <div className="relative grid place-items-start mt-4 md:mt-4 sm:mt-2 xs:mt-0 ml-[23rem] md:ml-[23rem] sm:ml-[19rem] xs:ml-[12rem]">
+              <ul className="list-disc list-inside text-pretty text-sm md:text-sm sm:text-xs xs:text-[9px]" style={{ lineHeight: "1" }}>
+                <li className="pl-[1rem]">Création de l'association Bab Rayan</li>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)